app/controlplane/pkg/biz/organization.go

```diff
@@ -244,3 +264,75 @@ func (uc *OrganizationUseCase) Delete(ctx context.Context, id string) error {
 	// Delete the organization
 	return uc.orgRepo.Delete(ctx, orgUUID)
 }
+
+// AutoOnboardOrganizations creates the organizations specified in the onboarding config and assigns the user to them
+// with the specified role if they are not already a member.
+func (uc *OrganizationUseCase) AutoOnboardOrganizations(ctx context.Context, userID string) error {
```

Comment:
I am wondering if it makes sense to pass the onboarding config as a parameter here instead. That way there is no dependency injection dependency in the module and can be tested more easily. But either way, it's fine :)